src/server/symbols.map

```diff
@@ -900,16 +908,678 @@ MIR_SERVER_2.17 {
+    virtual?thunk?to?mir::shell::ShellWrapper::surface_at*;
+    virtual?thunk?to?mir::shell::ShellWrapper::swap_z_order*;
+    vtable?for?mir::BasicCallback;
+    vtable?for?mir::DefaultServerStatusListener;
+    vtable?for?mir::DisplayChanger;
+    vtable?for?mir::DisplayServer;
+    vtable?for?mir::EmergencyCleanup;
+    vtable?for?mir::GLibMainLoop;
+    vtable?for?mir::LockableCallbackWrapper;
+    vtable?for?mir::ServerConfiguration;
+    vtable?for?mir::WaylandExtensionHook;
+    vtable?for?mir::compositor::BufferStream;
+    vtable?for?mir::compositor::ScreenShooter;
+    vtable?for?mir::detail::FdSources;
+    vtable?for?mir::detail::GMainContextHandle;
+    vtable?for?mir::detail::GSourceHandle;
+    vtable?for?mir::detail::SignalSources::HandlerElement;
+    vtable?for?mir::detail::SignalSources;
+    vtable?for?mir::frontend::BufferStream;
+    vtable?for?mir::frontend::Connector;
+    vtable?for?mir::frontend::DisplayChanger;
+    vtable?for?mir::frontend::DragIconController;
+    vtable?for?mir::frontend::EventSink;
+    vtable?for?mir::frontend::InputConfigurationChanger;
+    vtable?for?mir::frontend::PointerInputDispatcher;
+    vtable?for?mir::frontend::SurfaceStack;
+    vtable?for?mir::graphics::NullDisplayConfigurationObserver;
+    vtable?for?mir::input::KeyboardObserver;
+    vtable?for?mir::input::Scene;
+    vtable?for?mir::input::Seat;
+    vtable?for?mir::input::VTFilter;
+    vtable?for?mir::input::Validator;
+    vtable?for?mir::input::VirtualInputDevice;
+    vtable?for?mir::scene::Clipboard;
+    vtable?for?mir::scene::ClipboardObserver;
+    vtable?for?mir::scene::DataExchangeSource;
+    vtable?for?mir::scene::IdleHub;
+    vtable?for?mir::scene::IdleStateObserver;
+    vtable?for?mir::scene::NullObserver;
+    vtable?for?mir::scene::NullPromptSessionListener;
+    vtable?for?mir::scene::OutputProperties;
+    vtable?for?mir::scene::OutputPropertiesCache;
+    vtable?for?mir::scene::SceneChangeNotification;
+    vtable?for?mir::scene::SceneReport;
+    vtable?for?mir::scene::SessionContainer;
+    vtable?for?mir::scene::SessionEventHandlerRegister;
+    vtable?for?mir::scene::SessionEventSink;
+    vtable?for?mir::scene::SessionLock;
+    vtable?for?mir::scene::SessionLockObserver;
+    vtable?for?mir::scene::SurfaceEventSource;
+    vtable?for?mir::scene::SurfaceObservers;
+    vtable?for?mir::scene::SurfaceStateTracker;
+    vtable?for?mir::scene::TextInputChange::CursorPosition;
+    vtable?for?mir::scene::TextInputChange::TextInputKeySym;
+    vtable?for?mir::scene::TextInputChange::TextInputPreeditStyle;
+    vtable?for?mir::scene::TextInputChange;
+    vtable?for?mir::scene::TextInputChangeHandler;
+    vtable?for?mir::scene::TextInputHub;
+    vtable?for?mir::scene::TextInputState;
+    vtable?for?mir::scene::TextInputStateObserver;
+    vtable?for?mir::shell::DefaultWindowManager;
+    vtable?for?mir::shell::IdleHandler;
+    vtable?for?std::hash;
+  };
+local: *;
```

Comment:
I have a somewhat vague recollection that specifying the `local: *;` incantation in multiple stanzas is problematic. (I think it belongs in the primary one only)

Reply:
I'll fix this one on https://github.com/canonical/mir/pull/3305